charts-packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -967,23 +969,38 @@ export abstract class Chart extends Observable implements AgChartInstance {
             return;
         }
 
-        const pick = this.pickSeriesNode({ x: offsetX, y: offsetY }, !this.tooltip.tracking);
+        const isPixelRange = typeof range === 'number' && Number.isFinite(range);
+        const pick = this.pickSeriesNode(
+            { x: offsetX, y: offsetY },
+            range === 'exact',
+            // @ts-ignore Typescript doesn't recognise `range` must be a number here
+            isPixelRange ? range : undefined
+        );
```

Comment:
I'd avoid // @ts-ignore where possible - maybe try:
```suggestion
        let pixelRange = undefined;
        if (typeof range === 'number' && Number.isFinite(range)) {
          pixelRange = range;
        }
        const pick = this.pickSeriesNode(
            { x: offsetX, y: offsetY },
            range === 'exact',
            pixelRange,
        );
```

Reply:
Yeah, ts-ignore is not great. Only issue is I use `isPixelRange` again further down. But it does seem the only way to make ts happy is to directly wrap it in that `if` statement. We are a little behind on ts version but let's maybe save that for another day!

Will update as per your suggestion.